common/metrics/defs.go

```diff
@@ -788,6 +794,8 @@ const (
 	StorageLatencyTimer
 	// StorageWriteStoreLatency is the latency to write message to store
 	StorageWriteStoreLatency
+	// StorageWriteMessageLatency is the latency to write message to store and ack, but excludes time writing to socket
+	StorageWriteMessageExcludeSocketLatency
```

Comment:
@thuningxu @datoug added an additional new metric here. He aready un-commented the code above. That is, now there will be 3 metrics:
(1) pure rocksdb latency - StorageWriteStoreLatency 
(2) latency between the time store received the message and about to send the ack back to input (i.e, the *ExcludeSocket* latency). This is used to measure all the internal pump latency.
(3) StorageWriteMessageLatency - including the socket.